kubernetes/yamls/susi-server/susi-deployment.yml

```diff
@@ -0,0 +1,36 @@
+#
+# Deployment of the SUSI Server
+#
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: susi-server
+  namespace: web
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: susi-server
+    spec:
+      nodeSelector:
+        server: primary
+      containers:
+      - name: susi-server
+        image: chiragw15/susi_server2:latest
+        volumeMounts:
+          - mountPath: /data
```

Comment:
@singhpratyush This path should refer to the point where all data is saved, right? So, in susi, we have this `data` folder where all data in json files are located. So, it should be `/data`, right? or `/susi_server/data`?